code/game/antagonist/station/changeling.dm

```diff
@@ -17,7 +17,7 @@
 		"Bishop Accessory Frame"
 	)
 
-	welcome_text = "Use say \"#g message\" to communicate with your fellow changelings. Remember: you get all of their absorbed DNA if you absorb them."
+	welcome_text = "NOTICE: A changeling rework is in progress, and there may be new features or powers. Please report any bugs you experience. Use say \"#g message\" to communicate with your fellow changelings. Remember: you get all of their absorbed DNA if you perform a Full DNA Extraction them."
```

Comment:
Uncertain about the notice. There's a chance you won't finish the process, or we'll simply forget. At which point, this is going to get confusing.

Reply:
Alright, but it's not hard to remove, and helps people know it's being worked on.